src/components/publishing/header/author_date.tsx

```diff
@@ -0,0 +1,80 @@
+import moment from "moment-timezone"
+import * as React from "react"
+import styled, { StyledFunction } from "styled-components"
+import { pMedia } from "../../helpers"
+import Fonts from "../fonts"
+
+interface BulletTextProps extends React.HTMLProps<HTMLDivElement> {
+  layout?: string
+}
+
+interface AuthorDateProps {
+  authors: any
+  layout?: string
+  date?: string
+}
+
+const getAuthorByline = authors => {
+  if (authors) {
+    if (authors.length > 1) {
+      const str = authors.reduce((prev, curr, i) => {
+        let delim
+        const len = authors.length
+        if (i === len - 1) {
+          delim = " and "
+        } else if (i === 0) {
+          delim = ""
+        } else {
+          delim = ", "
+        }
+        return prev + delim + curr.name
+      }, "")
+      return str
+    } else {
+      return authors[0].name
+    }
+  } else {
+    return "Artsy Editorial"
+  }
+}
+
+const getDate = date => moment(date).tz("America/New_York").format("MMM D, YYYY h:mm a")
```

Comment:
Yea.. that's apparently what editorial wants now. Fwiw, this only affects editorial articles, since only editorial can have feature/standard headers.